### DISCUSSION TITLE
Freelancer

### DISCUSSION THREAD

**Discussion by rodriguesmichael** [dmroeder/pylogix#211] [General]
Hello all,
Someone here works as freelancer?

Basically, 6 screens (monitors) with information coming from a Logix 5000.

**Comment by TheFern2** [comment]
You might have better luck on r/PLC they post monthly posts where people can post jobs.
https://www.reddit.com/r/PLC/comments/za51ah/plc_jobs_classifieds_dec_2022/

**Comment by yichaoliang** [comment]
Hi Michael,

I am interested at this job.

May I know the detailed requirements and project schedule?

Regards
Chao

**Comment by TheFern2** [comment]
Locked out of topic, please use job boards in the future.
